jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/http/DocumentController.java

```diff
@@ -37,24 +38,23 @@
 public class DocumentController implements DocumentApi {
 	
 	private DocumentReadyHandler documentReadyHandler;
+	private AccessProperties accessProperties;
 	
 	/**
 	 * Creates a new document controller
 	 * @param documentReadyHandler the service that will handle the document
 	 */
-	public DocumentController(DocumentReadyHandler documentReadyHandler) {
+	public DocumentController(DocumentReadyHandler documentReadyHandler, AccessProperties accessProperties) {
 		
 		this.documentReadyHandler = documentReadyHandler;
-		
+		this.accessProperties = accessProperties;
 	}
 
 	/**
-	 * POST /document?sender={sender}
+	 * POST /document
 	 */
 	@SuppressWarnings({"unchecked", "rawtypes"})
-	@Override
-	public ResponseEntity<DocumentReceivedResponse> postDocument(@NotNull @Valid String sender,
-																 UUID xRequestID,
+	public ResponseEntity<DocumentReceivedResponse> postDocument(UUID xRequestID,
 																 UUID xB3TraceId,
```

Comment:
@override must be there, you need to generate the api in the documentApi module, the api is autogenerated from the swagger, you can document that in the readme as well of its not ;-)